### DISCUSSION TITLE
Accès aux param du REDY depuis SynApps

### DISCUSSION THREAD

**Discussion by Synfactory** [witsa/synapps#772] [Q&A] [answered]
Salut,
comment peut-on faire le lien de redirection via un bouton dans SynApps pour accéder aux paramétrages du REDY en _blank?Je partais sur l'idée d'un OnClick avec un script windows.open mais j'aimerai ne pas passer par la page de Log.

Merci

**Comment by WITNicolas** [comment]
Salut

Il faut reconstruire l'url globale en utilisant les liens additionnel
Domaine
WID

![image](https://github.com/user-attachments/assets/a1186311-9854-4230-9b71-6bddb7fc049f)


~~~~text
SYNAPPS-STUDIO-SCENE|{"config":{"key":"utlmterne","name":"URL Interne","additionalDefs":{"urlInterne":{"type":"text","helperTxt":"Chemin"}},"additionals":{"urlInterne":"/easy"}},"leadActor":{"type":"layout/stack","key":"stackRoot","properties":{"verticalAlignment":"expand","color":"black"},"children":[{"type":"display/html","key":"html1","properties":{"content":"<embed src={{host}}/WSID{{sid}}{{urlInterne}}?sscreen=1 width=100% height=100% type='application/pdf'/>","verticalAlignment":"expand"},"additionalDefs":{"host":{"type":"text"},"sid":{"type":"text"},"urlInterne":{"type":"text"}},"additionals":{"host":"192.168.1.31"},"bindings":{"additionals.sid":{"readMode":"onetime","source":"session@sid"},"additionals.host":{"readMode":"onetime","source":"host@domain"},"additionals.urlInterne":"stage@additionals.urlInterne"}}]}}

**Reply by Synfactory** [reply]
Merci Nicolas :)

**Reply by WITNicolas** [reply]
mise à jour je crois que j'ai répondu à côté de la plaque aprés mure reflexion, je pense que c'est ça que tu souhaitais faire

**Reply by Synfactory** [reply]
Ca m'affiche bien la racine du REDY mais les liens dans celui-ci ne fonctionnent. Il n'y a que les liens "Eploitation - PArametrage - Configuration" qui fonctionnent. Les sous-menu ne sont pas cliquables

**Reply by Synfactory** [reply, marked as answer]
C'est good, j'ai crée des additionelles et j'ai utilisé windows.open
```javascript
window.open(this.additionals.domaine+'/WSID'+this.additionals.utilisateur+'/easy','_blank')
``` 

Merci pour l'aide :)
